packages/blade-mcp/knowledgebase/components/BarChart.md

```diff
@@ -64,9 +64,9 @@ type ChartBarWrapperProps = {
    */
   colorTheme?: colorTheme;
   /**
-   * The layout of the bar chart.
+   * The orientation of the bar chart.
    */
-  layout?: 'horizontal' | 'vertical';
+  orientation?: 'horizontal' | 'vertical';
```

Comment:
we can go back to layout like we discussed in meeting since orientation and layout are 2 different things